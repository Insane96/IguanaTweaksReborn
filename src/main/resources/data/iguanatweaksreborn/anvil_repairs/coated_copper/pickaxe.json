{
	"item_to_repair": {
		"id": "iguanatweaksreborn:coated_copper_pickaxe"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:obsidian"
			},
			"amount": 3
		}
	]
}