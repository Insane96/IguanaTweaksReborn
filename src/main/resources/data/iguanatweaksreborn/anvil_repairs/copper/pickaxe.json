{
	"item_to_repair": {
		"id": "iguanatweaksreborn:copper_pickaxe"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:copper_ingot"
			},
			"amount": 3
		}
	]
}