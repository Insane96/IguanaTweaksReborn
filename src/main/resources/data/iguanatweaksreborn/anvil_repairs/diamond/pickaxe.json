{
	"item_to_repair": {
		"id": "minecraft:diamond_pickaxe"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:diamond"
			},
			"amount": 2
		}
	]
}