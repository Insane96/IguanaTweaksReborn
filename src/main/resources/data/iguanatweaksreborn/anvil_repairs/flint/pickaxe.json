{
	"item_to_repair": {
		"id": "iguanatweaksreborn:flint_pickaxe"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:flint"
			},
			"amount": 3
		}
	]
}