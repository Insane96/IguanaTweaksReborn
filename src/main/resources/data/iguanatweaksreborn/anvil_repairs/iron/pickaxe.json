{
	"item_to_repair": {
		"id": "minecraft:iron_pickaxe"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:iron_ingot"
			},
			"amount": 3
		}
	]
}