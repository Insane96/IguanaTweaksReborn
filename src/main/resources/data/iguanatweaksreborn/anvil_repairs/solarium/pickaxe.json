{
	"item_to_repair": {
		"id": "iguanatweaksreborn:solarium_pickaxe"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:iron_ingot"
			},
			"amount": 3,
			"max_repair": 0.5
		},
		{
			"repair_material": {
				"id": "iguanatweaksreborn:solarium_ball"
			},
			"amount": 2
		}
	]
}