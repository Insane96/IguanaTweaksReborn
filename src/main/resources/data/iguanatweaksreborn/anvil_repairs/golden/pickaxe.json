{
	"item_to_repair": {
		"id": "minecraft:golden_pickaxe"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:gold_ingot"
			},
			"amount": 3
		},
		{
			"repair_material": {
				"id": "minecraft:gold_nugget"
			},
			"amount": 27,
			"cost_multiplier": 9
		}
	]
}